### PR TITLE
Review_Session

### PR DESCRIPTION
**Pull request by fahimy143**
I am working on review of this project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+            href="https://fonts.googleapis.com/css?family=Poppins:200i,300,400&display=swap"
+            rel="stylesheet"
+           />
+          <link
+          href="https://fonts.googleapis.com/css?family=Raleway:700&display=swap"
+          rel="stylesheet"
+           />
+        </head>
+
+<body style="background-color: white;">
```
Comment (fahimy143) [inline index.html:25]: Please do not use inline style. its better to use it in separate CSS file.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+         />
+        <p class="project-title">
+        <span class="code">&lt;</span>
+        Hold a project 2
+        <span class="code">&#47;&gt;</span>
+        </p>
+        </a>
+    </div>
+   
+</section>
```
Comment (fahimy143) [inline index.html:83]: Consider making the outer wrapper a <section>, with each project as a div!

File: style.css [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+  }
+  
+  html {
+    box-sizing: border-box;
+  
+    font-size: 62.5%;
+    scroll-behavior: smooth;
+  }
+  
+
```
Comment (fahimy143) [inline style.css:26]: Kindly don't use extra spaces in your style!

**Review by fahimy143 — COMMENTED** [review]
@fahimy143 Nice Work! you did an amazing job so far. but still you have to work on some issues. you are almost don dud.

almost there


**Highlights**
Good Documentation ✔️
GitHub workflow is followed ✔️


**Required Changes**
Don't use extra spaces
Keep your style definition in a separate CSS file

Cheers and Happy coding!👏👏

Feel free to leave any questions or comments if something is not clear.

**Review by vechicin — APPROVED** [review]
# Hi @fahimy143  ✋, 

Your project is complete! 🎯 🧠 💯 
There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights ⚡ 

- [x] Fantastic use of inline comments.
- [x] Comments were very respectful and clear.

## Optional suggestions 📂 

- [ ] Keep in mind this exercise is designed to get you to perform a review as if you were a code reviewer. As such, you should be very mindful of grammar rules when performing reviews for other people because having glaring grammar mistakes will make your review look unprofessional. Your comments have some glaring grammar mistakes, I recommend you are more attentive to that moving forward.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**